aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ApplicationLoadBalancerRequestEvent.java

```diff
@@ -40,8 +41,8 @@
     private String path;
     private Map<String, String> queryStringParameters;
     private Map<String, List<String>> multiValueQueryStringParameters;
-    private Map<String, String> headers;
-    private Map<String, List<String>> multiValueHeaders;
+    private HttpHeaders<String> headers;
+    private HttpHeaders<List<String>> multiValueHeaders;
```

Comment:
I believe this is a breaking change for some consumers - I suggest we include it as part of the next major version of this library (v4). See https://github.com/aws/aws-lambda-java-libs/issues/282 where changes are being discussed